docs/getting_started/flask_example.py

```diff
@@ -44,4 +44,4 @@ def hello():
     return "hello"
 
 
-app.run(debug=True, port=5000)
+app.run(port=5000)
```

Comment:
running flask in debug mode on windows results in the server taking a very long time to start which makes the tests fail.